deps.edn

```diff
@@ -74,8 +74,6 @@
   hiccup/hiccup                             {:mvn/version "1.0.5"}              ; HTML templating
   inflections/inflections                   {:mvn/version "0.14.2"}             ; Clojure/Script library used for prularizing words
   instaparse/instaparse                     {:mvn/version "1.5.0"}              ; Make your own parser
-  junegunn/grouper                          {:mvn/version "0.1.1"}              ; Batch processing helper
-  clj-commons/clj-yaml                      {:mvn/version "1.0.29"}             ; Clojure wrapper for YAML library SnakeYAML
```

Comment:
I don't understand why someone did this... it's one thing to just ignore the request to keep things sorted and put something at the start or end... but why put it in a completely random spot? Active mess making just seems so much worse than passive mess making.